app/assets/v2/images/grants/hall_of_fame/envelope(350x200).svg

```diff
@@ -0,0 +1,2 @@
,"y":0.4}},{"t":300,"v":{"x":1,"y":1}},{"t":600,"v":{"x":1,"y":1}},{"t":1000,"v":{"x":0.4,"y":0.4}},{"t":1400,"v":{"x":1,"y":1}}]}},"opacity":[{"t":0,"v":0.46},{"t":400,"v":1},{"t":800,"v":0.51},{"t":1100,"v":1},{"t":1500,"v":0.46}]},"elrb9K7qzRH7":{"transform":{"data":{"o":{"x":239.200415,"y":56.000001,"type":"corner"},"t":{"x":-234.5,"y":-68}},"keys":{"s":[{"t":0,"v":{"x":0.5,"y":0.5}},{"t":500,"v":{"x":1,"y":1}},{"t":1000,"v":{"x":1.2,"y":1.2}},{"t":1500,"v":{"x":0.5,"y":0.5}}]}},"opacity":[{"t":0,"v":0.12},{"t":400,"v":1},{"t":900,"v":0},{"t":1500,"v":1}]},"elrb9K7qzRH9":{"transform":{"data":{"t":{"x":-179,"y":-101.787002}},"keys":{"o":[{"t":300,"v":{"x":179,"y":101.787003,"type":"corner"}},{"t":900,"v":{"x":178,"y":-37.787003,"type":"corner"}},{"t":1100,"v":{"x":179,"y":228.787002,"type":"corner"}},{"t":1500,"v":{"x":179,"y":101.787003,"type":"corner"}}],"r":[{"t":0,"v":0},{"t":130,"v":4},{"t":300,"v":0},{"t":1100,"v":0},{"t":1230,"v":4},{"t":1400,"v":0}]}},"opacity":[{"t":800,"v":1},{"t":900,"v":0},{"t":1000,"v":0},{"t":1090,"v":0},{"t":1100,"v":1}]},"elrb9K7qzRH13":{"transform":{"data":{"t":{"x":-179.072006,"y":-138.141502}},"keys":{"o":[{"t":0,"v":{"x":179.072006,"y":138.141502,"type":"corner"}},{"t":300,"v":{"x":179.072006,"y":141.141502,"type":"corner"}},{"t":700,"v":{"x":179.072006,"y":138.141502,"type":"corner"}}]}}},"elrb9K7qzRH14":{"transform":{"data":{"t":{"x":-179.074005,"y":-126.183498}},"keys":{"o":[{"t":0,"v":{"x":179.074005,"y":126.183498,"type":"corner"}},{"t":300,"v":{"x":179.074005,"y":130.183498,"type":"corner"}},{"t":700,"v":{"x":179.074005,"y":126.183498,"type":"corner"}}]}}},"elrb9K7qzRH15":{"transform":{"data":{"t":{"x":-187.498001,"y":-129.400002}},"keys":{"o":[{"t":0,"v":{"x":187.498001,"y":129.400002,"type":"corner"}},{"t":300,"v":{"x":187.498001,"y":126.400002,"type":"corner"}},{"t":700,"v":{"x":187.498001,"y":129.400002,"type":"corner"}}]}}},"elrb9K7qzRH16":{"transform":{"data":{"t":{"x":-187.498001,"y":-114.783001}},"keys":{"o":[{"t":0,"v":{"x":187.498001,"y":114.783001,"type":"corner"}},{"t":300,"v":{"x":187.498001,"y":112.783001,"type":"corner"}},{"t":700,"v":{"x":187.498001,"y":114.783001,"type":"corner"}}]}}},"elrb9K7qzRH17":{"transform":{"data":{"t":{"x":-171,"y":-132.264496}},"keys":{"o":[{"t":0,"v":{"x":171,"y":132.264496,"type":"corner"}},{"t":300,"v":{"x":171,"y":126.264496,"type":"corner"}},{"t":700,"v":{"x":171,"y":132.264496,"type":"corner"}}]}}},"elrb9K7qzRH18":{"transform":{"data":{"t":{"x":-171.001999,"y":-115.278999}},"keys":{"o":[{"t":0,"v":{"x":171.001999,"y":115.278999,"type":"corner"}},{"t":300,"v":{"x":171.001999,"y":112.278999,"type":"corner"}},{"t":700,"v":{"x":171.001999,"y":115.278999,"type":"corner"}}]}}}}}],"options":{"start":"hover","hover":"freeze","click":null,"scroll":null,"font":"embed","exportedIds":"unique","svgFormat":"animated","title":"envelope(350x200)"},"animationSettings":{"duration":1500,"direction":1,"iterations":1,"fill":1,"alternate":false,"speed":1}})]]></script></svg>
```

Comment:
to be more dry we moved the sdk to svgator_player.js and just load the instructions here. You can check other pages using svgator_player

Reply:
Ok, I will look into that.

Reply:
This has been implemented